File: www/src/types/CommissioningPackage.ts
import HighlightColors from "../enums/HighlightColors.ts";

export default interface CommissioningPackage {
  id: string;
  name: string;
  color: HighlightColors;
  boundaryNodes: NodeElement[];
  internalNodes: NodeElement[];
  selectedInternalNodes: NodeElement[];
}

export interface NodeElement {
  id: string;
}

File: client/Boundaries/Backend/Model/CommissioningPackage.cs
namespace Backend.Model;

public class CommissioningPackage
{
    public required string Id { get; set; }
    public required string Name { get; set; }
    public required string Color { get; set; }
    public List<Node>? InternalNodes { get; set; }
    public List<Node>? SelectedInternalNodes { get; set; }
    public List<Node>? BoundaryNodes { get; set; }

}
